src/components/routes/Host/HostPayments/HostPayments.tsx

```diff
@@ -141,6 +141,12 @@ class EnhancedComponent extends React.Component<Props, HostPaymentsContentState>
                 }
               </Button>
             </BeeLink>
+            {!stripeAccountDashboardLink && <BeeLink
+              href={`https://connect.stripe.com/oauth/authorize?response_type=code&client_id=${STRIPE_CLIENT_ID}&scope=read_write`}>
```

Comment:
can we hide this url inside a handler like `/account/stripe_express/new` (maybe /account/stripe_standard/connect or similar) so we can include it in emails too?